{"name": "test_login[Karina-2452]", "status": "failed", "statusDetails": {"message": "AssertionError: Invalid credentials\nassert 'Epic sadface... this service' == ''\n  \n  + Epic sadface: Username and password do not match any user in this service", "trace": "username = 'Karina', password = '2452'\n\n    @pytest.mark.login\n    @pytest.mark.parametrize(\"username, password\", get_data())\n    def test_login(username, password):\n        driver = SeleniumWebDriver(implicitly_wait=1, explicitly_wait=1)\n        page = SauceDemoPage(driver)\n        msg = page.Login(username, password)\n        driver.close(exit_time=2)\n>       assert msg == '', 'Invalid credentials'\nE       AssertionError: Invalid credentials\nE       assert 'Epic sadface... this service' == ''\nE         \nE         + Epic sadface: Username and password do not match any user in this service\n\nreports\\test_02_pytest_allure.py:25: AssertionError"}, "parameters": [{"name": "username", "value": "'Karina'"}, {"name": "password", "value": "'2452'"}], "start": 1707771393133, "stop": 1707771400079, "uuid": "dd6df58e-7f2e-48b7-bc50-c745f64ee680", "historyId": "7826494a61c155feaa70f4c19256f1c4", "testCaseId": "e30dcb273174ee6b93f1c89a0eff4e28", "fullName": "reports.test_02_pytest_allure#test_login", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "reports"}, {"name": "suite", "value": "test_02_pytest_allure"}, {"name": "host", "value": "DESKTOP-551JSCE"}, {"name": "thread", "value": "58820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "reports.test_02_pytest_allure"}]}